jobs/release-monitoring/release-monitoring-3scale/release-monitoring-3scale.yaml

```diff
@@ -0,0 +1,73 @@
+---
+- job:
+    name: 3scale-release-monitoring
+    display-name: 'Integreatly 3scale Release Monitoring'
+    project-type: pipeline
+    concurrent: false
+    triggers:
+      - timed: 'H */1 * * *'
+    parameters:
+      - string:
+          name: 'installationGitUrl'
+          default: 'git@github.com:odra/installation.git'
+          description: '[REQUIRED] The installation repo containing the components meta file (COMPONENTS.yaml)'
+      - string:
+          name: 'installationGitRef'
+          default: 'mestre'
+          description: '[REQUIRED] The installation git ref'
+      - string:
+          name: 'installationProductBranch'
+          default: '3scale-next'
+          description: '[REQUIRED] The installation git branch to push new version changes'
+      - string:
+          name: 'manifestVar'
+          default: 'threescale_version'
+          description: '[REQUIRED] The manifest variable to be used as the current component version'
+      - string:
+          name: 'projectOrg'
+          default: '3scale'
+          description: '[REQUIRED] github project organization'
+      - string:
+          name: 'projectRepo'
+          default: '3scale-amp-openshift-templates'
+          description: '[REQUIRED] github project repostirory'
+      - string:
+          name: 'githubToken'
+          default: 'jenkins-github-api-token'
+          description: '[REQUIRED] github api token secret text id to be used in its rest api requests'
+      - string:
+          name: 'credentialId'
+          default: 'jenkinsgithub'
+          description: '[REQUIRED] github ssh credential id to be used'
+      - string:
+          name: 'installJobName'
+          default: 'installation-pipeline'
+          description: '[REQUIRED] install job name to be used when a new version is available'
+      - bool:
+          name: 'dryRun'
+          default: false
+          description: '[REQUIRED] dryRun test flag'
+      - string:
+          name: 'clusterConfigCredentialsID'
+          default: '3scale-ga-cb40-config'
+          description: '[REQUIRED] json secret file id to be used'
+      - string:
+          name: 'testJobName'
+          default: 'installation-smoke-tests'
+          description: '[REQUIRED] smoke test project name'
+      - string:
+          name: 'uninstallJobName'
+          default: 'uninstallation-pipeline'
+          description: '[REQUIRED] uninstall project name'
+      
+    pipeline-scm:
+      script-path: jobs/release-monitoring/release-monitoring-3scale/Jenkinsfile
+      scm:
+        - git:
+            branches:
+              #- master
+              - 'release-monitoring'
+            #url: 'https://github.com/integr8ly/ci-cd.git'
+            url: 'https://github.com/odra/ci-cd.git'
```

Comment:
Needs changed to whatever it's supposed to be. You should also add a line into the configure_jenksin script to have it generated and pushed to the Jenkins instance https://github.com/integr8ly/ci-cd/blob/master/scripts/configure_jenkins.sh#L20

Reply:
Using it this way to test the changes and I will revert it back to master once the pr is in good shape

Reply:
Using the script thing i sent you it doesn't load from SCM, so you can make change locally and test it more easily that way.